### DISCUSSION TITLE
How to use custom error message ?

### DISCUSSION THREAD

**Discussion by Lenny4** [pestphp/pest#472] [Q&A]
Hello,

I can't find a way to write a custom error message when a test fail.

Something like:
```php

it('should not fail', function () {
    expect(false, 'My custom error message')->toBeTrue();
});
```

Thank you

**Comment by fabio-ivona** [comment]
As for now is not possible in a **simple** way, you could create a custom expectation or trigger a custom error message this way:

```php
it('should not fail', function () {
    try{
        expect(false)->toBeTrue();
    }catch(ExpectationFailedException $exception){
         throw new ExpectationFailedException('My custom error message');
    }
});
```

**Comment by Hambrook** [comment]
This is now possible, just with a different format.

```php
expect('foo')->toBe('foo', 'uh oh');
```

or for no-value expectations...

```php
expect(true)->toBe(true, 'oops');
```

Edit: @fabio-ivona, I just noticed it was you that added this support. Thanks!
